avro_schema/frontend.lua

```diff
@@ -113,7 +112,8 @@ local copy_schema_location_info
 -- forward_reference.
 local function scope_add_type(context, typeid, xtype)
     local scope = context.scope
-    local _, typeid = extract_nullable(typeid)
+    local _
+    _, typeid = extract_nullable(typeid)
```

Comment:
I don't think we should follow 'variable redefinition' warning of luacheck. I always disabled it in my projects. I don't ask for that here too.